### PR TITLE
Use fs-extra instead of wrench

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: tasks/test-coverage.js [hunk 1]
```diff
@@ -87,9 +73,9 @@ var setupBackupAndInstrumentationDir = function() {
   }
 
   log('• copy src files to backup folder');
-  wrench.copyDirSyncRecursive(dir, backupDir, copyOpts);
+  fse.copySync(dir, backupDir);
```
Comment (nicholas-l) [inline tasks/test-coverage.js]: This wont overwrite existing files does that matter? Also preserving timestamps? ```{clobber: true, preserveTimestamps: true}```

Comment (fredj) [inline tasks/test-coverage.js]: The backupDir should be empty at this time, but I will add the options to have the same as we had with wrench.

Comment (fredj) [inline tasks/test-coverage.js]: done

**Comment by ahocevar** [comment]
Looks good, please merge.

File: tasks/test-coverage.js [hunk 2]
```diff
@@ -8,8 +8,8 @@
  */
 
 var fs = require('fs');
+var fse = require('fs-extra');
```
Comment (marcjansen) [inline tasks/test-coverage.js]: You can just do

```javascript
var fs = require('fs-extra');
```

and remove the `fse` variable

File: tasks/test-coverage.js [hunk 3]
```diff
@@ -87,9 +80,9 @@ var setupBackupAndInstrumentationDir = function() {
   }
 
   log('• copy src files to backup folder');
-  wrench.copyDirSyncRecursive(dir, backupDir, copyOpts);
+  fse.copySync(dir, backupDir, copyOpts);
```
Comment (marcjansen) [inline tasks/test-coverage.js]: that'd be than `fs.copySync` also at the other places

**Comment by marcjansen** [comment]
The function `deleteFolderRecursive` can also be replaced with `fs.remove`.

If you could take my suggestions, @fredj, that'd be great. Otherwise looks good to me as well.

**Comment by fredj** [comment]
> The function `deleteFolderRecursive` can also be replaced with `fs.remove`.

done, thanks

File: tasks/test-coverage.js (hunk 2 above)
Comment (fredj) [inline tasks/test-coverage.js]: you're right but we have the `fs` and `fse` vars in all the others scripts in the tasks folder.
we can change that in an other PR

Comment (marcjansen) [inline tasks/test-coverage.js]: Agreed. :+1:

**Comment by marcjansen** [comment]
Thanks for the additional commit @fredj. Please merge when green.

**Comment by fredj** [comment]
thanks for the review

**Pull request merged**